addons/ui/functions/fnc_checkWeaponZeroing.sqf

```diff
@@ -0,0 +1,36 @@
+#include "..\script_component.hpp"
+/*
+ * Author: drofseh
+ * Display the zeroing of the player's current muzzle.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * Current Zero <NUMBER>
+ *
+ * Example:
+ * [ace_player] call ace_ui_fnc_checkWeaponZeroing
+ *
+ * Public: No
+ */
+
+params ["_unit"];
+
+private _vehicle = objectParent _unit;
+private _currentZero = -1;
+
+if (isNull _vehicle) then {
+    _currentZero = currentZeroing _unit;
+} else {
+    // In Arma3 v2.20 this will return the zero set by vehicle laser rangefinders
+    _currentZero = (_vehicle currentZeroing [_vehicle currentWeaponTurret (_vehicle unitTurret _unit), currentMuzzle _unit]) select 0;
```

Comment:
LHS should be the unit not the vic

```suggestion
    _currentZero = (_unit currentZeroing [_vehicle currentWeaponTurret (_vehicle unitTurret _unit), currentMuzzle _unit]) select 0;
```
Works in 2.20 for lasers (tested badger IFV)
Could also just do `currentZeroing player;`

Reply:
actually it's prob half borked
laser returns are raw numbers `685.026`
page up/down are arrays `[500,4]`

Reply:
using `param [0,0]` should fix